packages/web-app-files/tests/unit/components/SideBar/Shares/InviteCollaborator/InviteCollaboratorForm.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('InviteCollaboratorForm', () => {
+  describe('renders correctly', () => {
+    it.todo('renders a select field for share receivers')
```

Comment:
im fine with todos, just checking if its a leftover or if we will fix this later?

Reply:
Fix later;)